media/transcoder/src/main/java/com/example/transcoder/CreateJobFromPresetBatchMode.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.transcoder;
+
+// [START transcoder_create_job_from_preset_batch_mode]
+
+import com.google.cloud.video.transcoder.v1.CreateJobRequest;
+import com.google.cloud.video.transcoder.v1.Job;
+import com.google.cloud.video.transcoder.v1.Job.ProcessingMode;
+import com.google.cloud.video.transcoder.v1.LocationName;
+import com.google.cloud.video.transcoder.v1.TranscoderServiceClient;
+import java.io.IOException;
+
+public class CreateJobFromPresetBatchMode {
+
+  public static void main(String[] args) throws Exception {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project-id";
+    String location = "us-central1";
+    String inputUri = "gs://my-bucket/my-video-file";
+    String outputUri = "gs://my-bucket/my-output-folder/";
+    String preset = "preset/web-hd";
```

Comment:
Can we add a link to the docs which lists all preset types?

Reply:
Done